_posts/2023-07-28-ajaxmanager.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "[CVE-2023-33493] Unrestricted Upload of File with Dangerous Type in the Ajaxmanager File and Database explorer (ajaxmanager) module from RSI for PrestaShop"
+categories: module
+author:
+- Profileo.com
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ajaxmanager"
+severity: "critical (10)"
+---
+
+An "Unrestricted Upload of File with Dangerous Type" vulnerability exists in the Ajaxmanager File and Database explorer (ajaxmanager) module, from RSI, for PrestaShop, for all versions (including latest version 2.3.0). This allows remote attackers to upload dangerous files without restriction.
+
+## Summary
+
+* **CVE ID**: [CVE-2023-33493](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-33493)
+* **Published at**: 2023-07-28
+* **Advisory source**: Friends-Of-Presta
+* **Platform**: PrestaShop
+* **Product**: ajaxmanager
+* **Impacted release**: All versions (No fix provided. Still vulnerable in the latest version 2.3.0)
+* **Product author**: RSI
+* **Weakness**: [CWE-434](https://cwe.mitre.org/data/definitions/434.html)
+* **Severity**: critical (10)
+
+## Description
+
+In the Ajaxmanager File and Database explorer (ajaxmanager) module for PrestaShop, remote attackers can access a file explorer without being logged in, enabling upload view and deletion of files. The file explorer tool is also providing access to a shell console, port scan and server information. Disabling or uninstalling the module is not removing the access to the tool. The issue is not fixed in the latest version.
+
+It should be noted that the module provides users the ability to set a password to restrict access to the tool. However, the password is giving no protection. A bug allows users to access the file explorer without having to provide the password.
+
+This vulnerability has been successfully reproduced in the versions 2.1.0, 2.2.0 and 2.3.0 (the last version to date). We believe that the issue is also existing in previous versions.
```

Comment:
This vulnerability has been successfully reproduced in versions 2.1.0, 2.2.0 and 2.3.0 (the last version to date). We believe that the issue also existed in previous versions.